config/router-chatbot.js

```diff
@@ -34,10 +37,19 @@ router.post('/', (req, res) => {
     return app.locals.controllers.donorsChooseBot.chatbot(req, res);
   }
 
-  const mobilecommonsOip = process.env.CAMPAIGNBOT_MOBILECOMMONS_OIP;
-  if (!mobilecommonsOip) {
-    logger.error('CAMPAIGNBOT_MOBILECOMMONS_OIP undefined');
+  let configured = true;
+  // Check for required config variables.
+  const settings = ['MOBILECOMMONS_OIP_CAMPAIGNBOT', 'MOBILECOMMONS_OIP_AGENTVIEW'];
+  settings.push('MOBILECOMMONS_KEYWORD_CAMPAIGNBOT');
```

Comment:
Yeah this isn't as clean. I was debating on adding them all to 1 line adding using the `disable-eslint` -- would that be best @sergii-tkachenko ?